examples/bridge-app/linux/Device.cpp

```diff
@@ -110,6 +110,25 @@ void DeviceOnOff::SetOnOff(bool aOn)
     }
 }
 
+void DeviceOnOff::MQTTPublisher(mqtt::async_client& client, const std::string& topic, const std::string& payload) {
```

Comment:
Publish can be used by the device for all clusters.
```suggestion
void Device::MQTTPublisher(mqtt::async_client& client, const std::string& topic, const std::string& payload) {
```